### PR TITLE
Popup modal session

### PR DESCRIPTION
**Pull request by asharanjith**
In this session, 

- remove project cards from the index.html file 
- update project card using projects.js
- create a modal window using modals.js
- styles modal window using modals.css

### CODE REVIEW

File: modal.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+.modal-image-holder {
+  max-width: 80vw;
+}
+
+.modal-img {
+  min-width: 80vw;
+  max-height: 100vh;
+}
```
Comment (GzimAsani) [inline modal.css:52]: - [ ] (MOBILE) Add spacing on the bottom of the modal, on mobile version

Yours:
image
Design:
image

File: modals.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+            <img src="${modals[i].headerImgM}" alt="Project Sample Image" class="modal-img mobile">
+        </div>
+
+        
+            <div class="modal-desc">
+                
+                <div class="modal-article-container">
+                    <p class=" modal-para">${modals[i].descriptionItem}</p>
+                </div>
+                <div class="modal-tag-container">
```
Comment (GzimAsani) [inline modals.js:77]: - [ ] Add paddings on the tag container in order to match better the design

Yours:
image
Yours:
image

File: modal.css (hunk 1 above)
Comment (GzimAsani) [reply modal.css:52]: - [ ] (MOBILE) Add space between the name and the icon
image

Comment (GzimAsani) [reply modal.css:52]: - [ ] (MOBILE) Add this line on your modal, similar to the design
image

File: modal.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
+.modal {
```
Comment (GzimAsani) [inline modal.css:1]: ## DESKTOP:
- [ ] This looks different from the design:
Desing: 
image

Yours:
image

Comment (GzimAsani) [reply modal.css:1]: image

Comment (GzimAsani) [reply modal.css:1]: - [ ] These should be all centered:
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello    @asharanjith     :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with your setting up your javascript, it works good :+1:  
 
 ## Required Changes 

- [ ] Check the comment below :arrow_double_down: 






 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
Hi @asharanjith , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
goodjob

## Highlights
✔️ You're code is clean. well done 
✔️ your linters are all ok

## Optional suggestions


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**